src/main/resources/templates/fragments/entry-table.html

```diff
@@ -4,6 +4,10 @@
 <body>
 <div th:fragment="entry-table(content)">
     <table class="entry">
+        <colgroup>
+            <col class="field-column" />
```

Comment:
[No problems with self-closing tags in HTML5](http://www.w3.org/TR/html51/syntax.html#start-tags)